plugins/field-date/src/field_date.js

```diff
@@ -26,380 +28,378 @@ goog.require('goog.i18n.DateTimeSymbols');
-Blockly.Css.register([
-  /* eslint-disable indent */
-  '.blocklyDatePicker,',
-  '.blocklyDatePicker th,',
-  '.blocklyDatePicker td {',
-    'font: 13px Arial, sans-serif;',
-    'color: #3c4043;',
-  '}',
-
-  '.blocklyDatePicker th,',
-  '.blocklyDatePicker td {',
-    'text-align: center;',
-    'vertical-align: middle;',
-  '}',
-
-  '.blocklyDatePicker .goog-date-picker-wday,',
-  '.blocklyDatePicker .goog-date-picker-date {',
-    'padding: 6px 6px;',
-  '}',
-
-  '.blocklyDatePicker button {',
-    'cursor: pointer;',
-    'padding: 6px 6px;',
-    'margin: 1px 0;',
-    'border: 0;',
-    'color: #3c4043;',
-    'font-weight: bold;',
-    'background: transparent;',
-  '}',
-
-  '.blocklyDatePicker .goog-date-picker-previousMonth,',
-  '.blocklyDatePicker .goog-date-picker-nextMonth {',
-    'height: 24px;',
-    'width: 24px;',
-  '}',
-
-  '.blocklyDatePicker .goog-date-picker-monthyear {',
-    'font-weight: bold;',
-  '}',
-
-  '.blocklyDatePicker .goog-date-picker-wday, ',
-  '.blocklyDatePicker .goog-date-picker-other-month {',
-    'color: #70757a;',
-    'border-radius: 12px;',
-  '}',
-
-  '.blocklyDatePicker button,',
-  '.blocklyDatePicker .goog-date-picker-date {',
-    'cursor: pointer;',
-    'background-color: rgb(218, 220, 224, 0);',
-    'border-radius: 12px;',
-    'transition: background-color,opacity 100ms linear;',
-  '}',
-
-  '.blocklyDatePicker button:hover,',
-  '.blocklyDatePicker .goog-date-picker-date:hover {',
-    'background-color: rgb(218, 220, 224, .5);',
-  '}',
-  /* eslint-enable indent */
-]);
-
-Blockly.fieldRegistry.register('field_date', Blockly.FieldDate);
-
+Blockly.Css.register(`
+.blocklyDatePicker,
+.blocklyDatePicker th,
+.blocklyDatePicker td {
+  font: 13px Arial, sans-serif;
+  color: #3c4043;
+}
+
+.blocklyDatePicker th,
+.blocklyDatePicker td {
+  text-align: center;
+  vertical-align: middle;
+}
+
+.blocklyDatePicker .goog-date-picker-wday,
+.blocklyDatePicker .goog-date-picker-date {
+  padding: 6px 6px;
+}
+
+.blocklyDatePicker button {
+  cursor: pointer;
+  padding: 6px 6px;
+  margin: 1px 0;
+  border: 0;
+  color: #3c4043;
+  font-weight: bold;
+  background: transparent;
+}
+
+.blocklyDatePicker .goog-date-picker-previousMonth,
+.blocklyDatePicker .goog-date-picker-nextMonth {
+  height: 24px;
+  width: 24px;
+}
+
+.blocklyDatePicker .goog-date-picker-monthyear {
+  font-weight: bold;
+}
+
+.blocklyDatePicker .goog-date-picker-wday, 
+.blocklyDatePicker .goog-date-picker-other-month {
+  color: #70757a;
+  border-radius: 12px;
+}
+
+.blocklyDatePicker button,
+.blocklyDatePicker .goog-date-picker-date {
+  cursor: pointer;
+  background-color: rgb(218, 220, 224, 0);
+  border-radius: 12px;
+  transition: background-color,opacity 100ms linear;
+}
+
+.blocklyDatePicker button:hover,
+.blocklyDatePicker .goog-date-picker-date:hover {
+  background-color: rgb(218, 220, 224, .5);
+}
+`);
+
+Blockly.fieldRegistry.register('field_date', FieldDate);
+
+Blockly.FieldDate = FieldDate;
```

Comment:
This works only because we transpile Blockly to a non-ESM module, and is unlikely to work in future.  Is it not sufficient to add it to the registry?

Reply:
If someone is using it through `Blockly.FieldDate` they'll be broken if I delete this--but they'll be able to use it through the registry instead. That is, it won't disappear, but they will have to change code.

Reply:
Rachel, do you remember why you ended up deleting this? I think this did end up breaking people using `Blockly.FieldDate`.

Reply:
Adding properties directly to the Blockly module from outside of Blockly will eventually fail if/when we seal the module, and I thought the workaround (using it from the registry) was sufficient.